src/sentry/static/sentry/app/components/forms/apiForm.tsx

```diff
@@ -40,19 +41,36 @@ export default class ApiForm extends Form<Props> {
     this.api.clear();
   }
 
+  groupDataByDisabled() {
+    // Return two hashes, one with data for live elements and another for disabled.
+    const [{data}, disabledData] = [{...this.state}, {}];
+    this.props.children &&
+      this.props.children.forEach(child => {
+        // Bit of a hack: loop over components named *Field to find disabled ones.
+        if (child.type && child.type.name && RegExp('Field$').test(child.type.name)) {
```

Comment:
Why not the traditional regex notation?
```suggestion
        if (child.type && child.type.name && /Field$/.test(child.type.name)) {
```

Reply:
Yes! Done in fd7f5e346d4cc9589f58853e48815272ae378d01.